UDV-Core/src/Modules/Temporal/Utils/TransactionsManager.js

```diff
@@ -10,7 +10,6 @@ export class TransactionsManager {
          * @type {Array}
          */
         this.transactions = this.parseTransactions(transactions);
-
```

Comment:
`TransactionsManager.js` should not be modified by your PR.

Reply:
Cleaned